### ISSUE TITLE
Starred arguments are not correctly found on Python version < 3.5

### ISSUE THREAD

**Issue by jfinkels** [serge-sans-paille/gast#11]
On Python 3.5
```sh
$ python3.5 -c "import gast; print(gast.dump(gast.parse('f(*a)')))"
Module(body=[Expr(value=Call(func=Name(id='f', ctx=Load(), annotation=None), args=[Starred(value=Name(id='a', ctx=Load(), annotation=None), ctx=Load())], keywords=[]))])
```
On Python 3.4
```sh
$ python3.4 -c "import gast; print(gast.dump(gast.parse('f(*a)')))"
Module(body=[Expr(value=Call(func=Name(id='f', ctx=Load(), annotation=None), args=[], keywords=[]))])
```
There is a similar issue for kwargs.

**Comment by serge-sans-paille** [comment]
I'll have a look a t this, and do a release once fixed!

**Comment by jfinkels** [comment]
This was fixed by pull request #12.
